v2/contribute/decisions/session/0006-token-header-names-follow-cookie-names.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: "0006"
-title: Token header names should follow existing cookie names
```

Comment:
Also need a version of refreshing the session that doesn't depend on the req, res.